Src/CSharpier.Tests/TestFiles/InitializerExpression/InitializerExpressions.cst

```diff
@@ -62,8 +62,7 @@ class ClassName
         var collectionInitializerExpressions = new SomeObject
         {
             ThisIsACollection = { one, two },
-            ThisIsAlsoACollection =
-            {
+            ThisIsAlsoACollection = {
```

Comment:
I think we want to keep the old formatting on this, but your change does get this consistent with `arrayInitializerWithoutSize` above it. It could also be a new issue.